src/sql/workbench/electron-browser/modelComponents/diffeditor.component.ts

```diff
@@ -98,7 +120,7 @@ export default class DiffEditorComponent extends ComponentBase implements ICompo
 	}
 
 	private createUri(input: string): URI {
-		let uri = URI.from({ scheme: Schemas.untitled, path: `${this.descriptor.type}-${this.descriptor.id}-${input}` });
+		let uri = URI.from({ scheme: 'diffEditor', path: `${this.descriptor.type}-${this.descriptor.id}-${input}` });
```

Comment:
I wasn't sure what to put as the scheme here. Using Schemas.untitled returned an error and I wasn't sure if using another value in Schemas would affect other places because of the registerTextModelContentProvider() call above. @alanrenmsft  any advice on what should be used here?

Reply:
I would suggest something a bit more specific such as sqlDiffEditor

---
In reply to: [288806243](https://github.com/microsoft/azuredatastudio/pull/5703#discussion_r288806243) [](ancestors = 288806243)